src/internal/kopia/wrapper.go

```diff
@@ -277,6 +278,10 @@ func getItemStream(
 		encodeElements(itemPath.PopFront().Elements()...),
 	)
 	if err != nil {
+		if strings.Contains(err.Error(), "entry not found") {
+			err = errors.Wrap(ErrNotFound, err.Error())
+		}
```

Comment:
should we be using the error type in `internal/common/errors.go` here so we preserve any stack trace information the error may have?

Reply:
I don't think that's necessary here.  And we can always backtrack if needed.